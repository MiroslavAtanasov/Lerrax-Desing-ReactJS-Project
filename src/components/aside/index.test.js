import React from 'react'
import renderer from 'react-test-renderer'
import Aside from '.'
import TestingEnvironment from '../../test-utils/router'


describe('Aside component', () => {
    it('should render Aside component', () => {
        const tree = renderer.create(
            <TestingEnvironment value={{
                user: {
                    loggedIn: true,
                    id: '123'
                }
            }}>
                <Aside />
            </TestingEnvironment>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})