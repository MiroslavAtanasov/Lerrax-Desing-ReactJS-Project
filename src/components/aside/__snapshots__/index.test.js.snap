// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aside component should render Aside component 1`] = `
<aside
  className="container"
>
  <h1
    className="h1"
  >
    Featured Categories
  </h1>
  <div
    className="aside-link"
  >
    <a
      className="aside-a"
      href="/sofas"
      onClick={[Function]}
    >
      Sofas
    </a>
  </div>
  <div
    className="aside-link"
  >
    <a
      className="aside-a"
      href="/chairs"
      onClick={[Function]}
    >
      Chairs
    </a>
  </div>
  <div
    className="aside-link"
  >
    <a
      className="aside-a"
      href="/beds"
      onClick={[Function]}
    >
      Beds
    </a>
  </div>
  <div
    className="aside-link"
  >
    <a
      className="aside-a"
      href="/tables"
      onClick={[Function]}
    >
      Tables
    </a>
  </div>
  <div
    className="aside-link"
  >
    <a
      className="aside-a"
      href="/sectionals"
      onClick={[Function]}
    >
      Sectionals
    </a>
  </div>
  <div
    className="aside-link"
  >
    <a
      className="aside-a"
      href="/benches"
      onClick={[Function]}
    >
      Benches
    </a>
  </div>
  <div
    className="aside-link"
  >
    <a
      className="aside-a"
      href="/nightstands"
      onClick={[Function]}
    >
      Nightstands
    </a>
  </div>
  <div
    className="aside-link"
  >
    <a
      className="aside-a"
      href="/wardrobes"
      onClick={[Function]}
    >
      Wardrobes
    </a>
  </div>
  <div
    className="aside-link"
  >
    <a
      className="aside-a"
      href="/bookcases"
      onClick={[Function]}
    >
      Bookcases
    </a>
  </div>
  <div
    className="aside-link"
  >
    <a
      className="aside-a"
      href="/desks"
      onClick={[Function]}
    >
      Desks
    </a>
  </div>
</aside>
`;
